lib/sendgrid/mailer.ex

```diff
@@ -37,6 +35,7 @@ defmodule SendGrid.Mailer do
 
     case SendGrid.post(@mail_url, payload, [{ "Content-Type", "application/json" }]) do
       { :ok, %{ status_code: 202 } } -> :ok
+      { :ok, %{ status_code: 200 } } -> :ok
```

Comment:
I would prefer `{ :ok, %{ status_code: status_code } } when status_code in [200,202] -> :ok` instead.